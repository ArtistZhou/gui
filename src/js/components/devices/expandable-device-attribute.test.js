import React from 'react';
import renderer from 'react-test-renderer';
import ExpandableDeviceAttribute from './expandable-device-attribute';

describe('ExpandableDeviceAttribute Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ExpandableDeviceAttribute />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
