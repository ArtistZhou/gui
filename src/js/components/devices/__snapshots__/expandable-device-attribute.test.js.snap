// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  onClick={[Function]}
>
  <li
    className="MuiListItem-root-1 attributes MuiListItem-default-4 MuiListItem-gutters-9 MuiListItem-disabled-7 opaque"
    disabled={true}
  >
    <div
      className="MuiListItemText-root-13"
    >
      <div
        className="MuiTypography-root-19 MuiTypography-body1-28 MuiTypography-colorTextSecondary-52 MuiListItemText-secondary-17"
      >
        <div>
          <span
            className=" "
            style={Object {}}
          />
           
        </div>
      </div>
    </div>
  </li>
  <hr
    className="MuiDivider-root-55"
  />
</div>
`;
